cmd/geth/chaincmd.go

```diff
@@ -354,7 +383,97 @@ func exportChain(ctx *cli.Context) error {
 		}
 		err = utils.ExportAppendChain(chain, fp, uint64(first), uint64(last))
 	}
+	if err != nil {
+		utils.Fatalf("Export error: %v\n", err)
+	}
+	fmt.Printf("Export done in %v\n", time.Since(start))
+	return nil
+}
+
+func importHistory(ctx *cli.Context) error {
+	if ctx.Args().Len() != 1 {
+		utils.Fatalf("usage: %s", ctx.Command.ArgsUsage)
+	}
+
+	stack, _ := makeConfigNode(ctx)
+	defer stack.Close()
+
+	chain, db := utils.MakeChain(ctx, stack, false)
+	defer db.Close()
+
+	var (
+		start   = time.Now()
+		dir     = ctx.Args().Get(0)
+		network string
+	)
+
+	// Determine network.
+	if utils.IsNetworkPreset(ctx) {
+		switch {
+		case ctx.Bool(utils.MainnetFlag.Name):
+			network = "mainnet"
+		case ctx.Bool(utils.SepoliaFlag.Name):
+			network = "sepolia"
+		case ctx.Bool(utils.GoerliFlag.Name):
+			network = "goerli"
+		}
+	} else {
+		// No network flag set, try to determine network based on files
+		// present in directory.
+		var networks []string
+		for _, n := range params.NetworkNames {
+			entries, err := era.ReadDir(dir, n)
+			if err != nil {
+				return fmt.Errorf("error reading %s: %w", dir, err)
+			}
+			if len(entries) > 0 {
+				networks = append(networks, n)
+			}
+		}
+		if len(networks) == 0 {
+			return fmt.Errorf("no era1 files found in %s", dir)
```

Comment:
This will also print "no era1 files found" if they are under a different name. In my case I tried to run this pr on devnet-9 whcih will produce files like this: `unknown-00001-40745657.era1`. I can't import them anymore

Reply:
maybe we can add something s.th. if no other networks are found, it will try to import the blocks using the "unknown" network

Reply:
what about using the chain id as the prefix instead of the network name?  Or maybe something like `<vanity-network-name>-<chain-id>` where the vanity name _should_ match the chain id but it just warns if it doesn't?

Reply:
@MariusVanDerWijden: it is a fair argument that era1 should probably have a bit better support for unknown networks. However, we don't really have new unmerged networks these days. This makes me worried that people will attempt to do as you have done with devnet-9, export it as era1. We should really push people in non-pow networks to use regular era format as it has all the data relevant to the chain (e.g. the beacon data).

I looked a bit into more generic support for era1 to do what you were trying to do, but my feeling is the gate is actually kinda of useful. Let me know if you still disagree.

@ryanschneider: what will the chain id do for us in this case? Just a more robust identifier? As stated above, I don't think we need to generically support networks because the number of PoW networks we care about is dwindling (in a few years, only mainnet will be left).